docs/01-motivation/02-learning-goals.adoc

```diff
@@ -1,138 +1,150 @@
 === {learning-goals}
 
 // tag::DE[]
-[[LZ-1-1]]
-==== LZ 1-1: Qualitätsziele von flexiblen Architekturen
+.Der Aufbau des Kapitels folgt einer klaren Progression:
+
+- Erst das große Bild verstehen
```

Comment:
ist das für die finale Version gedacht? Oder nur als Notiz für euch?

Reply:
Der Lehrplan muss nicht/soll nicht die Struktur des Trainings vorgeben. Lediglich zusammengehörige Punkte sollten pro Kapitel gesammelt sein.

Reply:
Wir haben es in Kapitel 00b-basis verschoben